utils/frame/remote-externalities/Cargo.toml

```diff
@@ -14,7 +14,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0" }
-env_logger = "0.9"
+tracing-subscriber = { version = "0.3.16", features = ["env-filter"] }
```

Comment:
This is only being used in tests?

Reply:
yupp,  this is just library but it used from the try-runtime CLI IIRC.

Reply:
If you look at your changes, you only call this code in tests? Or do I miss something?

Reply:
yes, you are right it's only used in tests good catch I thought it was only a dev-dependency already

I'll change that.